src/packaging/docker-services/reaper-ssl/reaper-ssl.env

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# use the credentials that match the ./docker/cassandra/jmxremote.* configurations
+REAPER_JMX_USERNAME=reaperUser
+REAPER_JMX_PASSWORD=reaperPass
+
+# define the Dockerized Cassandra node to serve as the Reaper backend
+# while using the reaper_db keyspace
+REAPER_STORAGE_TYPE=cassandra
+REAPER_CASS_CLUSTER_NAME="reaper-cluster"
+REAPER_CASS_CONTACT_POINTS=["cassandra-ssl"]
+REAPER_CASS_KEYSPACE=reaper_db
+
+# use SSL encryption when connecting to Cassandra via the native protocol
```

Comment:
Only include what's different between `reaper` and `reaper-ssl`.

Reply:
Ok. Done.